### ISSUE TITLE
stratum2

### ISSUE THREAD

**Issue by saiaman** [andru-kun/wildrig-multi#256]
Hello stratum mining.hello doesn't respect :
https://eips.ethereum.org/EIPS/eip-1571

You submit an array when an object is required

**Comment by andru-kun** [comment]
yeah, that one is not implemented right. But what are you trying to mine with it?
